conf/config.json

```diff
@@ -0,0 +1,544 @@
+    "RestrictSystemAdmin": false,
+    "UseNewSAMLLibrary": false,
+    "CloudUserLimit": 0,
+    "CloudBilling": false,
+    "EnableSharedChannels": false
+  },
+  "AnalyticsSettings": {
+    "MaxUsersForStatistics": 2500
+  },
+  "ElasticsearchSettings": {
+    "ConnectionUrl": "http://localhost:9200",
+    "Username": "elastic",
+    "Password": "changeme",
+    "EnableIndexing": false,
+    "EnableSearching": false,
+    "EnableAutocomplete": false,
+    "Sniff": true,
+    "PostIndexReplicas": 1,
+    "PostIndexShards": 1,
+    "ChannelIndexReplicas": 1,
+    "ChannelIndexShards": 1,
+    "UserIndexReplicas": 1,
+    "UserIndexShards": 1,
+    "AggregatePostsAfterDays": 365,
+    "PostsAggregatorJobStartTime": "03:00",
+    "IndexPrefix": "",
+    "LiveIndexingBatchSize": 1,
+    "BulkIndexingTimeWindowSeconds": 3600,
+    "RequestTimeoutSeconds": 30,
+    "SkipTLSVerification": false,
+    "Trace": ""
+  },
+  "BleveSettings": {
+    "IndexDir": "",
+    "EnableIndexing": false,
+    "EnableSearching": false,
+    "EnableAutocomplete": false,
+    "BulkIndexingTimeWindowSeconds": 3600
+  },
+  "DataRetentionSettings": {
+    "EnableMessageDeletion": false,
+    "EnableFileDeletion": false,
+    "MessageRetentionDays": 365,
+    "FileRetentionDays": 365,
+    "DeletionJobStartTime": "02:00"
+  },
+  "MessageExportSettings": {
+    "EnableExport": false,
+    "ExportFormat": "actiance",
+    "DailyRunTime": "01:00",
+    "ExportFromTimestamp": 0,
+    "BatchSize": 10000,
+    "DownloadExportResults": false,
+    "GlobalRelaySettings": {
+      "CustomerType": "A9",
+      "SmtpUsername": "",
+      "SmtpPassword": "",
+      "EmailAddress": "",
+      "SMTPServerTimeout": 1800
+    }
+  },
+  "JobSettings": {
+    "RunJobs": true,
+    "RunScheduler": true
+  },
+  "PluginSettings": {
+    "Enable": true,
+    "EnableUploads": false,
+    "AllowInsecureDownloadUrl": false,
+    "EnableHealthCheck": true,
+    "Directory": "./plugins",
+    "ClientDirectory": "./client/plugins",
+    "Plugins": {},
+    "PluginStates": {
+      "com.mattermost.nps": {
+        "Enable": true
+      }
+    },
+    "EnableMarketplace": true,
+    "EnableRemoteMarketplace": true,
+    "AutomaticPrepackagedPlugins": true,
+    "RequirePluginSignature": false,
+    "MarketplaceUrl": "https://api.integrations.mattermost.com",
+    "SignaturePublicKeyFiles": []
+  },
+  "DisplaySettings": {
+    "CustomUrlSchemes": [],
+    "ExperimentalTimezone": true
+  },
+  "GuestAccountsSettings": {
+    "Enable": false,
+    "AllowEmailAccounts": true,
+    "EnforceMultifactorAuthentication": false,
+    "RestrictCreationToDomains": ""
+  },
+  "ImageProxySettings": {
+    "Enable": false,
+    "ImageProxyType": "local",
+    "RemoteImageProxyURL": "",
+    "RemoteImageProxyOptions": ""
+  },
+  "CloudSettings": {
+    "CWSUrl": "https://customers.mattermost.com"
+  },
+  "FeatureFlags": {
+    "TestFeature": "off",
+    "CloudDelinquentEmailJobsEnabled": false
+  }
+}
```

Comment:
Mattermosts's configuration file is large, and changes quite often (almost with each release). This makes me skeptical on copying the full config file in Yunohost (because it's going to be out-to-date soon), rather than patching the vendor-provided config file at install-time.

That's not blocking for merging this PR – but patching the config file at runtime seems more sensible to me.